eng/Versions.props

```diff
@@ -106,6 +106,12 @@
     <!-- Prohibit the usage of .NET Standard 1.x dependencies. -->
     <FlagNetStandard1XDependencies Condition="'$(DotNetBuildSourceOnly)' == 'true'">true</FlagNetStandard1XDependencies>
   </PropertyGroup>
+
+  <!-- SolutionPersistence -->
+  <PropertyGroup>
+    <MicrosoftVisualStudioSolutionPersistenceVersion>1.0.28</MicrosoftVisualStudioSolutionPersistenceVersion>
```

Comment:
Revert changes to this file. I believe this is only being used for package which we consume through Arcade subscriptions.